Automattic/jetpack
# Search standalone plugin prototyping

#### Changes proposed in this Pull Request:
The PR changes all necessary parts to  make search standalone plugin work. After we make sure everything is working or figure out what we need to work on. Then small PRs will be proposed. Traditional customizer will be support thru another PR.

#### Jetpack product discussion
pcNPJE-Ch-p2

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
- Build the package and the plugin: 
  - `jetpack build packages/search`
  - `jetpack build packages/my-jetpack`
  - `jetpack build plugins/search`
  - `jetpack build plugins/jetpack`
- Navigate to you local dev instance, deactivate and disconnect Jetpack
- Navigate to `/wp-admin`
- Ensure `Search` menu is added as sub menu to Jetpack
- Click `Search` sub menu
- Ensure you could connect the site by clicking `Login`
- Ensure Dashboard works
- Ensure Customberg works
- Ensure frontend search overlay works
- Ensure JP search widget works
- Enable Jetpack plugin
- Ensure Jetpack Search still works
- Ensure you could manage search on My Jetpack page.

![image](https://user-images.githubusercontent.com/1425433/155442422-3e9facd0-8bd0-4134-bacc-1c6efb3d1905.png)
